besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1197,39 +1188,6 @@ static class PermissionsOptionGroup {
           "Sets target gas limit per block. If set, each block's gas limit will approach this setting over time if the current gas limit is different.")
   private final Long targetGasLimit = null;
 
-  // Tx Pool Option Group
-  @CommandLine.ArgGroup(validate = false, heading = "@|bold Tx Pool Options|@%n")
```

Comment:
I'd like to keep this grouping for ux. I get your point about the unstable options - can we keep the unstable options on the TransactionPoolOptions class (they don't appear in the help anyway) and put the remaining tx-pool options within this group in BesuCommand

Reply:
updated